pkg/juicefs/juicefs.go

```diff
@@ -687,15 +684,11 @@ func (j *juicefs) ceFormat(secrets map[string]string, noUpdate bool, setting *co
 		envs = append(envs, "JFS_NO_CHECK_OBJECT_STORAGE=1")
 	}
 	formatCmd.SetEnv(envs)
-	var res []byte
-	_, err := util.DoWithinTime(context.TODO(), 50*time.Second, nil, func() (err error) {
```

Comment:
We don't add timeout for `ctx`, let's keep this for this PR.

Reply:
We should keep these change in #409

Reply:
Changes reverted